concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1144,6 +1144,34 @@ jobs:
 
                   cf push
 
+        - task: sync-admin-users
+          config:
+            platform: linux
+            image: docker:///ruby#2.2
```

Comment:
@dcarley @saliceti I think we probably _should_ revisit it in another story, possibly one about pipeline speedup improvements.